core/shared/src/main/scala/zio/NonEmptyOps.scala

```diff
@@ -16,6 +18,9 @@ trait NonEmptyOps[+A, CC[+_], EC[+_]] {
   def map[B](f: A => B): CC[B]
   def reduce[B >: A](op: (B, B) => B): B
   def tail: EC[A]
+  def toArray[B >: A: ClassTag]: Array[B] = toList.toArray
+  def toIterable: Iterable[A]             = toList
+  def toList: List[A]                     = iterator.toList
```

Comment:
Same as above, I think `toIterable.toArray` might be better for cases that collection uses List as the implementation